apps/widget/src/app/FlaskCommunication.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import axios from "axios";
+
+import { config } from "./config";
+
+export async function callFlaskService<TRequest, TResponse>(
```

Comment:
Do we want to match file name with function name inside?
That should be easier to maintain.